README.md

```diff
@@ -7,6 +7,7 @@ This collection has been tested and supports MSO 2.1+.
 Modules supporting new features introduced in MSO API in specific MSO versions might not be supported in earlier MSO releases.
 
 *Note: This collection is not compatible with versions of Ansible before v2.8.*
+*Note: The Nexus Dashboard (ND) HTTPAPI connection plugin should be used when Cisco ACI Multi-Site is installed on Nexus Dashboard.*
```

Comment:
Would it be worth mentioning that all MSO versions post 3.2 need to use the 
ND plugin?